elements/pfe-navigation/src/pfe-navigation-item.js

```diff
@@ -0,0 +1,429 @@
+import PFElement from "../pfelement/pfelement.js";
+
+if (!("path" in Event.prototype)) {
+    Object.defineProperty(Event.prototype, "path", {
+      get: function() {
+        var path = [];
+        var currentElem = this.target;
+        while (currentElem) {
+          path.push(currentElem);
+          currentElem = currentElem.parentElement;
+        }
+        if (path.indexOf(window) === -1 && path.indexOf(document) === -1)
+          path.push(document);
+        if (path.indexOf(window) === -1)
+          path.push(window);
+        return path;
+      }
+    });
+}
+
+if (!Array.prototype.filter){
```

Comment:
I'm guessing we're making these files larger than they need to be with this `Array.prototype.filter` check. The same check is in pfe-navigation.js so this code is going to be duplicated. Can we remove this one and just rely on the one in pfe-navigation.js?

Reply:
It will not upgrade without the polyfill manually added.  If you can fix the polyfill tool, that would be rad but in the meantime, without these in here, it won't upgrade in Edge/IE11.